lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -251,8 +253,17 @@ combineLsqData NodePoolLsqData{nOpt, rewards, stake} =
         , saturation = (sat s)
         }
 
-    rewardsButNoStake = traverseMissing $ \k r ->
-        error $ "Rewards but no stake: " <> show (k, r)
+    -- TODO: This case seems possible on shelley_testnet, but why, and how
+    -- should we treat it?
+    --
+    -- The pool with rewards but not stake didn't seem to be retiring.
```

Comment:
Could it be because the leader schedule is based on the stake from the previous epoch? So a pool can have 0 stake now, but had some in the previous epoch and therefore, is getting rewards in this epoch.

Reply:
> So a pool can have 0 stake now

I think that I confirmed that pools in the stake distribution may have 0 stake. I.e. if this pool is entirely _missing_, something else is going on.